src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,12 +1,6 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
```

Comment:
I beleive you should not remove no-args constructor and you should Add some custom fields